tests/integration/golang/database/import_test.go

```diff
@@ -58,37 +58,72 @@ func (s *ImportTestSuite) SetupTest() {
 	assert.Nil(s.T(), database.CreateDefaultExperiment(db.GormDB(), "s3://fasttrackml"))
 	s.inputDB = db.GormDB()
 
-	inputExperimentFixtures, err := fixtures.NewExperimentFixtures(db.GormDB())
-	assert.Nil(s.T(), err)
-	inputRunFixtures, err := fixtures.NewRunFixtures(db.GormDB())
+	inputRunFixtures, err := fixtures.NewRunFixtures(s.inputDB)
 	assert.Nil(s.T(), err)
 	s.inputRunFixtures = inputRunFixtures
+	s.populateDB(s.inputDB)
```

Comment:
Does this test specifically demonstrate the fix to the reported problem? If it's must my inability to grok these verbose golang tests then just point it out and tell me to go away. :)